### ISSUE TITLE
Change column type enum to support more databases

### ISSUE THREAD

**Issue by goetzal** [anchan828/typeorm-helpers#679]
Hey @anchan828,

thanks for your extension. Is it possible to change the column type for history entity from enum to varchar?

Benefit of that change would be the support of more database systems. In my case I use mssql /azure sql and sqlite.
I tested it already and it works prety good but I'm not sure if there are any additional dependencies with that.

**Comment by anchan828** [comment]
Hi @goetzal,

you mean you want to change `action` column type from enum to varchar, right?
I also regret a bit that I used enum😢.  I will try that `synchronize: true` can safely migrate from enum to varchar.

For now, you can use vachar by overriding columOption.

```ts
@HistoryActionColumn({ type: "varchar", enum: undefined })
```

**Comment by goetzal** [comment]
Exactly. 

Thanks for your hint. Didn't thought about that.

**Comment by anchan828** [comment]
`synchronize: true` executes two queries (drop and add again). This will cause all the data in the action column to be lost and use the default value of CREATE.

<img width="607" alt="af5b0f684e433176894c750f0624f8ee" src="https://user-images.githubusercontent.com/694454/129872019-5b2064e2-faa7-486e-ac4a-ef1d7cea19e8.png">

I'd like to believe that no app is using `synchronize: true` in a production environment, but... It destroys data, so I'd be cautious.

**Comment by goetzal** [comment]
Then I woudln't change it either. Maybe you can add the overwriting line to the readme. Would have saved me some headache :).

Thanks for your support and time.

**Comment by anchan828** [comment]
Hi @goetzal 

Updated readme: https://github.com/anchan828/typeorm-helpers/tree/master/packages/history#overwrite-action-column-type
And published v0.6.6 https://www.npmjs.com/package/@anchan828/typeorm-history/v/0.6.6

You can override it by simply specifying the type. `enum: undefined` is not required.
```ts
@HistoryActionColumn({ type: "varchar" })
```
